src/NHSD.BuyingCatalogue.Ordering.Api/Extensions/OrderExtensions.cs

```diff
@@ -57,10 +58,13 @@ public static bool IsAssociatedServicesSectionComplete(this Order order)
             return !(order is null) && order.AssociatedServicesViewed;
         }
 
-        public static bool IsSectionStatusComplete(this Order order, int serviceRecipientsCount, int catalogueSolutionsCount, int associatedServicesCount)
+        public static bool IsSectionStatusComplete(this Order order, int serviceRecipientsCount)
         {
             if (!IsFundingSourceComplete(order)) return false;
```

Comment:
I didn't know we were doing single-line `if` statements now? 🤢

Reply:
Hah, it's a brave new world I guess